trustfall_wasm/src/trustfall_wasm.d.ts

```diff
@@ -51,6 +51,17 @@ export interface Adapter<T> {
 
 export class Schema {
     free(): void;
+    /**
+     * Returns a Set<string> which contains every subtype of type_name
+     * in the schema and itself. This allows you to get the subtypes using
+     * this function, then in resolveCoercion return whether the set returned
+     * by this function has the typename string of the vertex from the iterator,
+     * which would mean coercible from it's current form to the form wanted by
+     * resolveCoercion.
```

Comment:
Good function descriptions are hard, let's try to keep them concise while still communicating necessary information. Here's a suggestion:
```suggestion
     * Returns a `Set<string>` naming all subtypes (plus itself) of `type_name` in the schema.
     *
     * May be used in `Adapter.resolveCoercion` to determine if vertices can be coerced
     * to the specified type.
```

Reply:
This is great, thanks!